### DISCUSSION TITLE
Plotting speed.

### DISCUSSION THREAD

**Discussion by chiaXXL** [Chia-Network/chia-blockchain#2191] [General]
Hello,
Im just starting farming and was surprised that creating a new plot is extremely slow.
I have top hardware like intel optane ssd and core i9 CPU. And creating 1 plot doesnt utilize it more than 20%
Where is a bottle neck?  For example if I setup extremely high speed server with 768 GB of RAM and put temp directory into the RAM disk will it faster? Or it is just a waste of time and there is limitations from Chia wallet?
I have 2 PB storage and fill it with current speed will took years.

**Comment by deuko** [comment]
when creating a plot check "advanced settings". You can assign more working RAM and Cores to the plotting process.

![image](https://user-images.githubusercontent.com/4406412/115391414-bb794d00-a1df-11eb-90c9-afdf7cd0760b.png)

**Reply by chiaXXL** [reply]
Do I need to change number of buckets?
Im surrently testing 32 gb ram usage 32 threads and 128 buckets.

**Reply by deuko** [reply]
According to this page, you don't want / need to change bucket number:
https://www.chia.net/2021/02/22/plotting-basics.html

**Reply by dorkmo** [reply]
you might read about parralel plotting

**Reply by chiaXXL** [reply]
I will try to build 80 virtual machines on SSD using 10x3.2 TB enterprise ssd and 26x12TB storage.
This should give me 3 plots x 80 = 240 plots per day.

**Reply by wanghongqi8548488** [reply]
I can draw a picture to increase the number of threads and speed faster than the default. My cpu is 12 cores and 24 threads

**Reply by xillkoi** [reply]
my 5950x is very slow,why

**Comment by pubyun** [comment]
1. RAM size
2. SSD DISK size
3. cores of CPU

然后就是并发数量，可以加微信 refactor 探讨

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
